third_party/envoy-gateway/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN mkdir -p /var/lib/eg
 FROM ${CALICO_BASE}
 
 ARG TARGETARCH
+ARG GIT_VERSION=unknown
```

Comment:
Would you like this answer to be better than "unknown"?  I can probably help with that, if you can tell me what format the answer needs to be in.

(Same for the other envoy images.)

Reply:
assigning "unknown" follows the same pattern as the other components. Its values gets replaced with the `GIT_VERSION` when the image is built.

Reply:
Ah, nice, thanks for explaining that.